src/electron/views/results/test-view.tsx

```diff
@@ -34,21 +34,25 @@ export const TestView = NamedFC<TestViewProps>('TestView', props => {
     const { title, description } = contentPageInfo;
     const headerSection = <HeaderSection title={title} description={description} />;
 
+    if (contentPageInfo.instancesSectionComponent == null) {
```

Comment:
An alternative might be to use the null pattern and remove this case.

Basically, just pass a null component for the instances section and eliminates the optional parameter and logic from the UI.

Reply:
We specifically do have one called `NullComponent`.

Reply:
I think that might be better (and then in the future moving the spinner inwards to the instances section/something specific to where it applies).